### DISCUSSION TITLE
Consider open sourcing SSO

### DISCUSSION THREAD

**Discussion by davispuh** [SigNoz/signoz#6604] [Ideas]
I consider SSO to be essential feature even for us sole users so that we don't need additional password but instead use OIDC like Keycloak/Authentik.
And I don't think anyone would really switch to enterprise version just because of this single feature.
So please open source SSO to be available in community version.

**Comment by davispuh** [comment]
https://ssotax.org/

**Comment by Nageshbansal** [comment]
Hey @davispuh, our latest release of SigNoz Community Edition features SSO support (Google OAuth) and API key management. 
For full details, see the release notes at: https://github.com/SigNoz/signoz/releases/tag/v0.85.0.

You can also check out the blog discussing the new release: https://signoz.io/blog/open-source-signoz-now-available-with-sso-and-api-keys/

**Reply by AB027PS** [reply]
Hi @Nageshbansal, considering SSO is available in the community edition, are you considering making SAML available in the community edition as well?
